### PR TITLE
Recommended monitor doc

### PR DESCRIPTION
**Pull request by hestonhoffman**
Depends on https://github.com/DataDog/integrations-core/pull/15791
Wait for that one to be merged before merging this or the builds will break

Before merging, switch the pull config back to `master` (or the preview builds will break) and `cache_enabled` back to `true`

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.
- [ ] Check that `cache_enabled` is set to `true` in the `pull_config_preview.yaml` file

### CODE REVIEW

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 1]
```diff
@@ -96,14 +96,14 @@
...
-          branch: master
+          branch: eho1307/integrations-create-recmonitor
           globs:
           - docs/dev/*.md
           options:
             dest_dir: '/developers/integrations/'
             path_to_remove: 'docs/dev/'
             front_matters:
-              dependencies: [ "https://github.com/DataDog/integrations-core/blob/master/docs/dev/" ]
+              dependencies: [ "https://github.com/DataDog/integrations-core/blob/eho1307/integrations-create-recmonitor/docs/dev/" ]
```
Comment (hestonhoffman) [inline local/bin/py/build/configurations/pull_config_preview.yaml:106]:
```suggestion
              dependencies: [ "https://github.com/DataDog/integrations-core/blob/master/docs/dev/" ]
```

File: local/bin/py/build/configurations/pull_config_preview.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 - config:
-    cache_enabled: true
+    cache_enabled: false
```
Comment (hestonhoffman) [inline local/bin/py/build/configurations/pull_config_preview.yaml:3]:
```suggestion
    cache_enabled: true
```

**Review by estherk15 — COMMENTED** [review]
@hestonhoffman do you want the changes committed? Or are they placeholders? 
NVM just read your comment!

**Comment by hestonhoffman** [comment]
> @hestonhoffman do you want the changes committed? Or are they placeholders? NVM just read your comment!

Yeah, let's keep them until we're done previewing any changes in the other PR. When we're ready to merge we can commit that stuff

**Review by estherk15 — APPROVED** [review]

**Pull request merged**